cypress/e2e/signIn.cy.js

```diff
@@ -10,29 +9,37 @@ const homePage = new HomePageObject();
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
```

Comment:
The database clear function and user generation are now being executed for each test case. If your test cases do not modify the database or the user, consider moving this back to a 'before' hook to improve test performance.